test/index.test.js

```diff
@@ -67,12 +59,14 @@ describe('Element', () => {
 beforeAll(async () => {
     phantomjsProcess = await phantomjs.run('--webdriver=4444');
     await testApp.start();
-    await session.start();
-    await session.url('http://localhost:8087');
+    session = await newSession('http://localhost:4444', {
+        browserName: 'Chrome'
```

Comment:
I was just wondering is this browser name still valid here after you changed to the phantom JS approach?